src/domains/checkout-app/api/checkout/checkout_auth_service/v1/_openapi.json.tpl

```diff
@@ -337,10 +379,6 @@
       "UserInfoResponse": {
         "type": "object",
         "properties": {
-          "userId": {
```

Comment:
why removing here the userId field?
This is object for another PR, please restore here